app/controllers/admin/categories_controller.rb

```diff
@@ -10,6 +10,19 @@ def show
       }
     end
 
+    def new
+      render "admin/categories/new", locals: {
+        category: Category.new
+      }
+    end
+
+    def create
+      category_params = params.require(:category).permit(:name)
```

Comment:
Organization nit: it's generally nice to tuck this into a private method like we do in [`PiecesController`](https://github.com/mikekavouras-organize/art/blob/master/app/controllers/admin/pieces_controller.rb#L52)

Reply:
yes, daddy